test/unit/ui/app/actions.spec.js

```diff
@@ -534,7 +528,7 @@ describe('Actions', () => {
         { type: 'DISPLAY_WARNING', value: 'error' },
       ]
 
-      connectHardwareSpy.callsFake((deviceName, page, hdPath, callback) => {
+      connectHardwareSpy.callsFake((_, __, ___, callback) => {
```

Comment:
Nit: I like using `_` to denote "this is not used, please ignore", but to have 3 variations seems... strange?

Reply:
Well we need to get to the 4th argument. I'd rather not have so many args straight up ignored but instead of rewriting this test to use them, I'm just ignoring them. (Also, as a side note, the only allowed name for a unused argument is `/[_]+/` in our ESLint config.)